corehq/apps/app_manager/models.py

```diff
@@ -3258,7 +3258,7 @@ def create_profile(self, is_odk=False, with_media=False, template='app_manager/p
             'uniqueid': self.copy_of or self.id,
             'name': self.name,
             'descriptor': u"Profile File"
-        }).decode('utf-8')
+        }).encode('utf-8')
```

Comment:
According to @emord, `render_to_string` returns [`SafeText`](https://docs.djangoproject.com/en/1.5/ref/utils/#django.utils.safestring.SafeText), which is a subclass of `unicode`. So yes, this has been broken all along, and only worked because the result of `render_to_string` could be automatically converted to ASCII bytes before being `decode('utf-8')`d back to unicode.

I asked @emord to verify that this function (`create_profile`) is expected to return bytes, even though up to this point it had been returning a unicode object, which incidentally could be converted to ASCII without errors.